server/cmwell-it/src/it/scala/cmwell/it/AggregationTests.scala

```diff
@@ -0,0 +1,104 @@
+/**
+  * Copyright 2015 Thomson Reuters
+  *
+  * Licensed under the Apache License, Version 2.0 (the “License”); you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *   http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+  * an “AS IS” BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  *
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+
+package cmwell.it
+
+import com.typesafe.scalalogging.LazyLogging
+import org.scalatest.{AsyncFunSpec, Inspectors, Matchers}
+import play.api.libs.json.{JsValue, _}
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+import scala.util.{Failure, Success, Try}
+
+class AggregationTests extends AsyncFunSpec with Matchers with Inspectors with Helpers with fixture.NSHashesAndPrefixes with LazyLogging {
+
+  describe("Agg API should") {
+    //Assertions
+    val ingestGeonames = Future.traverse(Seq(293846)) { n =>
+      val agg = scala.io.Source.fromURL(this.getClass.getResource(s"/agg/aggnames_$n.nq")).mkString
+      Http.post(_in, agg, Some("text/nquads;charset=UTF-8"), List("format" -> "nquads"), tokenHeader)
+    }.map { results =>
+      forAll(results) { res =>
+        withClue(res) {
+          res.status should be(200)
+          jsonSuccessPruner(Json.parse(res.payload)) shouldEqual jsonSuccess
+        }
+      }
+    }
+
+
+    val path = cmw / "test.agg.org" / "Test201903_05_1501_11" / "testStatsApiTerms"
+
+    val aggForIntField = executeAfterCompletion(ingestGeonames) {
+      spinCheck(100.millis, true)(Http.get(
+        uri = path,
+        queryParams = List("op" -> "stats", "format" -> "json", "debug-info" -> "", "ap" -> "type:term,field::num.testns,size:3"))) { r =>
```

Comment:
you should use the full URI as preparation for distributed system tests
e.g. `num.$http://qa.test.rfnt.com/v1.1/testns$`